src/main/java/seedu/address/logic/commands/CountryNoteEditCommand.java

```diff
@@ -109,8 +112,4 @@ public boolean equals(Object other) {
         return targetIndex.equals(c.targetIndex) && countryNote.equals(c.countryNote) && tags.equals(c.tags);
     }
 
-    @Override
-    public int hashCode() {
```

Comment:
Why is this hashcode method deleted?

Reply:
All the other command classes don't have the hashcode function, so I deleted this one to standardize